.github/workflows/run-vsphere-tests.yaml

```diff
@@ -77,3 +77,18 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
           SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+      - name: Docker cleanup
+        if: always()
+        run: |
+          CONTAINERS=$(docker ps --all --quiet)
```

Comment:
can it somehow cause problems for e2e suites running in QA environment?

Reply:
QA is using another runner so this doesn't have any impact.